### PR TITLE
Han Wang Task2&4

### PR DESCRIPTION
**Pull request by han-wang19**
Task2: Created a class TankHanWang.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Project 2: Battle Game
+<!-- Interesting! -->
```
Comment (qinchuanhui) [inline README.md:2]: You should not change the code here, which I guess is just a test at first.

**Review by qinchuanhui — CHANGES_REQUESTED** [review]
You just inherit the original tank and do nothing🤦‍♀️. You need to design and implement **your own** tank functions.

File: README.md (hunk 1 above)
Comment (han-wang19) [reply README.md]: Yes it's a test and I have changed it back.

**Comment by han-wang19** [comment]
Task4: Created a new tank RightHandFastTank, solving the issue #387.

**Comment by han-wang19** [comment]
> You just inherit the original tank and do nothing🤦‍♀️. You need to design and implement **your own** tank functions.

I modified TankHanWang so that it can fire the bullet Missile instead of CannonBall, and it can move faster than TinyTank.

**Comment by han-wang19** [comment]
Task4: Created a new tank AirplaneHanWang, solving the issue #394.

**Review by qinchuanhui — APPROVED** [review]
LGTM，you can get the corresponding score

**Pull request closed without merging**